thirdeye-server/src/main/java/ai/startree/thirdeye/service/AlertService.java

```diff
@@ -127,6 +124,11 @@ protected void deleteDto(final AlertDTO dto) {
 
   @Override
   protected void prepareCreatedDto(final ThirdEyePrincipal principal, final AlertDTO dto) {
+    final String currentNamespace = authorizationManager.currentNamespace(principal);
```

Comment:
in prepareCreatedDto, the alert dto must have its namespace already set correctly. 
so you can use 
```
dto.namespace()
```
directly

Reply:
done